App_Data/SwaggerSpecs/Microsoft.ServiceFabric/managedcluster.json

```diff
@@ -0,0 +1,1232 @@
+{
```

Comment:
Since these are new files we also need to include them both in the csproj similar to
https://github.com/projectkudu/AzureResourceExplorer/blob/d0d6e17b33b6cf3d81ec94c9885ce92a0113a009/ARMExplorer.csproj#L395

Reply:
Updated with the new commit, thanks!